docs/reference/api/docker_remote_api_v1.21.md

```diff
@@ -1290,7 +1294,8 @@ Status Codes:
           "playdate:latest"
         ],
         "Size": 0,
-        "VirtualSize": 2429728
+        "VirtualSize": 2429728,
+        "Labels": {}
```

Comment:
interesting; did you actually get `{}` here? I just tried on 1.9.0-dev and 1.8.2 and got `"Labels":null`